sb3_contrib/common/maskable/algorithms/base.py

```diff
@@ -0,0 +1,86 @@
+from typing import Optional, Tuple
+
+import numpy as np
+from stable_baselines3.common.base_class import BaseAlgorithm
+from stable_baselines3.common.callbacks import BaseCallback, CallbackList, ConvertCallback
+from stable_baselines3.common.type_aliases import MaybeCallback
+from stable_baselines3.common.vec_env import VecEnv, is_vecenv_wrapped
+
+from sb3_contrib.common.maskable.callbacks import MaskableEvalCallback
+from sb3_contrib.common.maskable.policies import MaskablePolicy
+from sb3_contrib.common.vec_env.wrappers import VecActionMasker
+from sb3_contrib.common.wrappers import ActionMasker
+
+
+class MaskableAlgorithm(BaseAlgorithm):
```

Comment:
you don't need to define a new base class, you can overwrite `wrap_env` in the PPO algorithm only

Reply:
Wouldn't it be helpful to have a base class that could be used for other algorithms besides PPO? Even if that's out of scope right now, is that not something worth supporting as a possibility down the line?

Reply:
I was thinking along the lines of other masked algorithms just within this repo, but it's hard to argue with starting simple :grinning: